### DISCUSSION TITLE
Boostrap 5 - any plans?

### DISCUSSION THREAD

**Discussion by ITspirit** [dachcom-digital/pimcore-toolbox#221] [General]
Boostrap 5 - any plans on supporting this Theme or is maybe a PR welcome?

**Comment by solverat** [comment]
Hey @ITspirit, since we're moved away from bootstrap, it's not on our roadmap. If you're willing to provide a proper PR for it, we would accept it, of course.

- add new section in `templates/toolbox/bootstrap5`
- add calculators in `src/Calculator/Boostrap5`
- add identifier in `LayoutManagerInterface`
- add docs to `docs/themes/Bootstrap5.md`
